src/util/taskUtil.c

```diff
@@ -54,3 +54,8 @@ size_t ticksToMs(size_t ticks)
 {
     return ticks * portTICK_RATE_MS;
 }
+
+void yield()
```

Comment:
cool. shipit after that

On Sat, Jan 30, 2016 at 4:55 PM, Andrew Stiegmann <notifications@github.com>
wrote:

> In src/util/taskUtil.c
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/406#discussion_r51353432>
> :
>
> > @@ -54,3 +54,8 @@ size_t ticksToMs(size_t ticks)
> >  {
> >      return ticks * portTICK_RATE_MS;
> >  }
> > +
> > +void yield()
>
> Agreed. Though I had pulled it. Will nuke from orbit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/406/files#r51353432>
> .
>



-- 
Brent Picasso
*Autosport Labs <http://www.autosportlabs.com>*
*Technology for Race and Street*